apollo/typeDefs/user.ts

```diff
@@ -10,3 +10,9 @@ export const user = gql`
     user(id: String!): User!
   }
 `;
+
+export const createUser = gql`
+  extend type Mutation {
+    createUser(username: ID!, password: String): String
```

Comment:
i think this will be structured differently - idk if instead of `ID` here you would just use `String`

Reply:
Probably a string here, ID is processed as a string, but usually not human readable

Reply:
gotcha